components/LocationComparisonSection/LocationComparisonChartSection.tsx

```diff
@@ -28,6 +30,11 @@ const LocationComparisonChartSection: FunctionComponent<ComponentProps> = props
     if (options.indicator) {
       setSelections(options);
       setQuery(options, props.locations);
+      addEvent('locationComparisonOptionsChanged', {
+        topic: options.theme?.name,
+        indicator: options.indicator.name,
+        chartNumber: props.chartNumber
```

Comment:
We don't need this. We only need to check on the topic & indicator... 
![image](https://user-images.githubusercontent.com/5672438/80206585-40fe0780-8635-11ea-86bb-22e3af1282b2.png)
As you can see, tracking chart number splits the analytics based on chart count, which is a stat we don't care about ... We just want to know which topic & indicator.

Reply:
Updated to ignore the chart count